### ISSUE TITLE
Installation instructions: setting up AppArmor on Ubuntu 20.04 fails

### ISSUE THREAD

**Issue by priojk** [DoliCloud/SellYourSaas#228]
**Describe the bug**

Following the installation instructions for AppArmor [as described here](https://github.com/DoliCloud/SellYourSaas/blob/master/doc/Documentation%20SellYourSaas%20-%20Master%20and%20Deployment%20Servers%20-%20EN.asciidoc#setup-of-apparmor) fails.

The previously created files containing lines starting with 'link:' cannot be processed and the following errors are stated:

`deploy1 apparmor.systemd[5800]: AppArmor parser error for /etc/apparmor.d in /etc/apparmor.d/usr.bin.secureBash at line 1: syntax error, unexpected TOK_LINK, expecting $end`

`deploy1 apparmor.systemd[5832]: AppArmor parser error for /etc/apparmor.d/php-fpm7.4 in /etc/apparmor.d/php-fpm7.4 at line 1: syntax error, unexpected TOK_LINK, expecting $end`

`deploy1 apparmor.systemd[5870]: AppArmor parser error for /etc/apparmor.d/usr.bin.secureBash in /etc/apparmor.d/usr.bin.secureBash at line 1: syntax error, unexpected TOK_LINK, expecting $end`

**To Reproduce**

1. Set up AppArmor accrodning to the instructions, especially creating the described files all starting with link:
2. Reload AppArmor with the command `aa-teardown; /etc/init.d/apparmor restart;`
    result is stated as failed
3. Checking the failure with `systemctl status apparmor.service` results in the errors mentioned above.

**Expected behavior**

Successful reload of AppArmor after configuring it.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 on OVH public cloud

**Comment by hregis** [comment]
@priojk 
and with ?

`systemctl restart apparmor.service`

**Comment by priojk** [comment]
Thanks for taking the time to look into this.

I tried your change, but the AppArmor parser failed again. However, I think you pointed me into the right direction. Looking up the manual from AppArmor, I came up with the following line instead:

`#include "/home/admin/wwwroot/dolibarr_sellyoursaas/etc/apparmor.d/bin.secureBash"`

using #include instead of include::, inserting what I guess might be the document_root and removing the [ ] at the end.

_(I am wondering, if this line should have been processed by a tool like ansible instead of directly with the apparmor parser?)_

When I do this for all three files (secureBash, apache2 and php-fpm7.4) the parsing does work ok. But there is still an error caused by the contents of the linked file in php-fpm7.4:

` AppArmor parser error for /etc/apparmor.d in /home/admin/wwwroot/dolibarr_sellyoursaas/etc/apparmor.d/php-fpm7.4 at line 16: Could not open 'local/php-fpm7.4'`

The referenced line has the following contents:
`  #include <local/php-fpm7.4>`

I noticed that in the French documentation, there are no includes used but the contents is directly copied into the files. But I guess the English document is slightly more up to date than the French one?
